### ISSUE TITLE
Can't work with Caltech101

### ISSUE THREAD

**Issue by lynnporu** [pytorch/vision#4904]
### 🐛 Describe the bug

When I try to download and use Caltech101 dataset in Google Colab, I get a following error:

```python
Extracting datasets/caltech101/101_ObjectCategories.tar.gz to datasets/caltech101

---------------------------------------------------------------------------

OSError                                   Traceback (most recent call last)

/usr/lib/python3.7/tarfile.py in gzopen(cls, name, mode, fileobj, compresslevel, **kwargs)
   1645         try:
-> 1646             t = cls.taropen(name, mode, fileobj, **kwargs)
   1647         except OSError:

                                     <15 frames>

OSError: Not a gzipped file (b'vV')


During handling of the above exception, another exception occurred:

ReadError                                 Traceback (most recent call last)

/usr/lib/python3.7/tarfile.py in gzopen(cls, name, mode, fileobj, compresslevel, **kwargs)
   1648             fileobj.close()
   1649             if mode == 'r':
-> 1650                 raise ReadError("not a gzip file")
   1651             raise
   1652         except:

ReadError: not a gzip file
```

I use this code:

```python
from torchvision import datasets
caltech_df = datasets.Caltech101("datasets", download=True)
```

### Versions

PyTorch version: 1.9.0+cu111
Is debug build: False
CUDA used to build PyTorch: 11.1
ROCM used to build PyTorch: N/A

OS: Ubuntu 18.04.5 LTS (x86_64)
GCC version: (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0
Clang version: 6.0.0-1ubuntu2 (tags/RELEASE_600/final)
CMake version: version 3.12.0
Libc version: glibc-2.26

Python version: 3.7.12 (default, Sep 10 2021, 00:21:48)  [GCC 7.5.0] (64-bit runtime)
Python platform: Linux-5.4.104+-x86_64-with-Ubuntu-18.04-bionic
Is CUDA available: False
CUDA runtime version: 11.1.105
GPU models and configuration: Could not collect
Nvidia driver version: Could not collect
cuDNN version: Probably one of the following:
/usr/lib/x86_64-linux-gnu/libcudnn.so.7.6.5
/usr/lib/x86_64-linux-gnu/libcudnn.so.8.0.5
/usr/lib/x86_64-linux-gnu/libcudnn_adv_infer.so.8.0.5
/usr/lib/x86_64-linux-gnu/libcudnn_adv_train.so.8.0.5
/usr/lib/x86_64-linux-gnu/libcudnn_cnn_infer.so.8.0.5
/usr/lib/x86_64-linux-gnu/libcudnn_cnn_train.so.8.0.5
/usr/lib/x86_64-linux-gnu/libcudnn_ops_infer.so.8.0.5
/usr/lib/x86_64-linux-gnu/libcudnn_ops_train.so.8.0.5
HIP runtime version: N/A
MIOpen runtime version: N/A

Versions of relevant libraries:
[pip3] numpy==1.19.5
[pip3] torch==1.9.0+cu111
[pip3] torchsummary==1.5.1
[pip3] torchtext==0.10.0
[pip3] torchvision==0.10.0+cu111
[conda] Could not collect

**Comment by malfet** [comment]
I can reproduce it using torchvision-0.10, but it has been fixed in 0.11, so adding `!pip install torchvision==0.11.1` cell fixes the problem

**Comment by lynnporu** [comment]
Well, your comment will be useful for someone with the same trouble :smile:
